spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/AbstractAdaptableMessageListener.java

```diff
@@ -104,6 +123,28 @@ public void setResponseExchange(String responseExchange) {
 		this.responseExchange = responseExchange;
 	}
 
+	/**
+	 * Set the default replyTo address to use when sending response messages.
+	 * This is only used if the replyTo from the received message is null.
+	 * <p>
+	 * Response destinations are only relevant for listener methods
+	 * that return result objects, which will be wrapped in
+	 * a response message and sent to a response destination.
+	 * <p>
+	 * Can be a string starting with "SpEL:" in which case the expression is
+	 * evaluated at runtime; see the reference manual for more information.
+	 * @param defaultReplyTo The exchange.
+	 * @since 1.6
+	 */
+	public void setResponseAddress(String defaultReplyTo) {
+		if (defaultReplyTo.startsWith("SpEL:")) {
```

Comment:
That smells a bit.
How about to use:
```
PARSER.parseExpression(defaultReplyTo, new TemplateParserContext());
```
instead?
And rely on the standard SpEL syntax: `#{myBean.foo}` etc.